src/main/cpp/t04_both2.cpp

```diff
@@ -23,9 +23,37 @@
 
 #include "t04_both2.h"
 #include <iostream>
+#include <set>
 
 using namespace std;
 
 int t04_both2() {
-
-}
+    set <int>z;
+    set <int>z2;
+    set <int>z1;
+    int q;
+    int w;
+    cin >> q;
+    for(int i = 0; i < q; i++)
+    {
+        int a;
+        cin >> a;
+        z.insert(a);
+    }
+    int k = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'k' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4688993)